oximeter/instruments/src/kstat/link.rs

```diff
@@ -221,15 +226,17 @@ mod tests {
     }
 
     #[test]
-    fn test_physical_datalink() {
+    fn test_sled_datalink() {
         let link = TestEtherstub::new();
-        let sn = String::from("BRM000001");
```

Comment:
so, this was outdated, but there were no checks against the current fake-serial/gimlet settings so that any string would have worked?

Reply:
If I follow your question, then yes -- this is a fake value used only in the tests. It's now populated from the const `SLED_SERIAL` above.